Cargo.toml

```diff
@@ -31,7 +31,11 @@ fluent-bundle = "0.15"
 fluent-langneg = "0.13"
 fluent-syntax = "0.11"
 fluent-templates = "0.13"
+h2 = "0.4"
+hickory-resolver = "0.25"
 http = "1.3"
+hyper = "0.14.32" # Use the same version as `reqwest`
```

Comment:
As discussed on Slack, we are pinning this to the version used by the current `reqwest` version rather than using `*`.